src/oc_erchef/apps/oc_chef_wm/src/oc_chef_action_insights.erl

```diff
@@ -0,0 +1,67 @@
+%% Copyright 2012-2014 Chef Software, Inc. All Rights Reserved.
```

Comment:
guess that means I copied-wrong ![Caruso](http://esq.h-cdn.co/assets/cm/15/07/54d90d754880e_-_david-caruso-0409-lg.jpg)